### PR TITLE
Added support for ap-northeast-2

### PR DESCRIPTION
**Pull request by stevemac007**
Added support for the new Seoul region from endpoints publicaly listed at http://docs.aws.amazon.com/general/latest/gr/rande.html

### CODE REVIEW

File: boto/endpoints.json [hunk 1]
```diff
@@ -410,8 +434,9 @@
     },
     "sts": {
         "ap-northeast-1": "sts.amazonaws.com",
-        "ap-southeast-1": "sts.amazonaws.com",
-        "ap-southeast-2": "sts.amazonaws.com",
+        "ap-northeast-1": "sts.ap-northeast-2.amazonaws.com",
```
Comment (kyleknap) [inline boto/endpoints.json]: Let's stick with using the global endpoint here to be consistent with the rest of the regions.

**Comment by kyleknap** [comment]
Thanks for the PR. I just had one comment. Once you update, I will go ahead and merge it.

**Comment by stevemac007** [comment]
All updated and ready for a merge.   When is the next release planned for?  Would love to use this ASAP.

**Comment by kyleknap** [comment]
Thanks! We are trying to cut a release soon especially since we have quite a few features merged into develop and a release has not happened for a while now.

**Pull request merged**